test/service/test_request_validation.py

```diff
@@ -87,11 +88,28 @@ def tearDown(self):
         if self.server_thread.is_alive():
             self.fail('Thread is still alive after joining')
 
-    def test_hello_world(self):
-        url = self.base_url
+    def test_health_es_reachable(self):
+        url = self.base_url + "health"
         response = requests.get(url)
+        expected_response = {
+            'status': 'UP',
+            'elasticsearch': {'domain': AzulConfig().es_domain, 'status': 'UP'}
+        }
         self.assertEqual(response.status_code, 200, response.json())
-        self.assertEqual({"Hello": "World!"}, response.json())
+        self.assertEqual(response.json(), expected_response)
+
+    def test_health_es_unreachable(self):
+        self.env = EnvironmentVarGuard()
+        self.env.set('AZUL_ES_ENDPOINT', 'nonexisting-index.us-east-1.es.amazonaws.com:80')
+
+        url = self.base_url + "health"
+        response = requests.get(url)
+        expected_response = {
+            'status': 'UP',
+            'elasticsearch': {'domain': AzulConfig().es_domain, 'message': 'Unable to reach the host', 'status': 'DOWN'}
+        }
+        self.assertEqual(response.status_code, 200, response.json())
```

Comment:
This is bad. Please see my comment in the other PR.

Use `response.raise_for_status()`